docs/Course Wiki/CS Course Offerings/CS225.md

```diff
@@ -0,0 +1,90 @@
+
+CS225 also has around 12 labs which are smaller weekly programming questions.  They have in person meetings which are optional, but often contain a smaller intro to the lab and allow you to make groups to work on the labs.  Each of the labs can be worked on with a group up to 10 people, and can be submitted as many times as you want before the due date on PrairieLearn. The purpose of labs is to help improve your programming abilities and reinforce concepts taught in lecture. Exact lab formats may change week-to-week, but a usual lab will consist of about 20 minutes of discussion over the material covered in lecture and about 90 minutes of collaborative coding. Lab assignments will be released for all students before the first lab on Wednesday, and will be due the following Sunday at 11:59pm.
+
+### Exams
+
+CS225 has 6 smaller exams and a final, with the smaller exams being 50 minutes long and the final being 110 minutes long.  They typically consist of a few multiple choice and short answer questions, and one programming question.  All of the exams are taken through the CBTF, so are scheduled through their website.
+
+### Grading
+
+Category |	Contribution| Notes 
+--- | --- | ---
+Programming Assignments |	360 points |	60 points each
+Laboratory Assignments |	120 points |	10 points each
+Exams | 360 points |	60 points each
+Final Exam |	160 points |
+**Total** | **1000 points**
+
+|Extra Credit (capped at 100 points) | |
+|--- | --- | 
+Early MP submission | 40 points 
+Problem of the Day | 40 points 
+Final Project | Up to 60 points 
+
+## Instructors
+
+Currently, Carl Evans is the primary Instructor for CS 225 with Brad Solomon teaching the CS225 Honors Section of the Course.  
+
+## Course Tips
+
+CS 225 often offers extra credit for submitting a part of the MP early.  This can be very useful for increasing your grade and is relatively easy given you start early on the MPs.  Starting early and working at a steady pace for this class can be very helpful in ensuring your success for this class, as leaving the MPs for the last day or few days can be a large task.
+
+Another source of extra credit in CS225 is the Problem of the Days, which are assigned daily on PrairieLearn starting the second week. Each problem completed counts as 1 point of extra credit.  These can be extremely useful to learn the C++ syntax, especially for ECE students who only had a few weeks to learn C++ in ECE220.
+
+A final project is also offered for extra credit at the end of the semester to work on in a group.  You typically make a project utilizing some of the algorithms learned in the class, and details are posted towards the end of the semester.
```

Comment:
change first sentence to "A group final project is offered for extra credit at the end of the semester."

Reply:
second sentence to "Projects utilize algorithms and data structures covered in the class, and exact details and requirements are posted toward the end of the semester."